fvm/src/call_manager/mod.rs

```diff
@@ -121,6 +121,12 @@ pub trait CallManager: 'static {
         self.gas_tracker_mut().apply_charge(charge)?;
         Ok(())
     }
+
+    /// Whether tracing is requested.
+    fn tracing(&self) -> bool;
```

Comment:
Any reason not to just call `call_manager.machine().context().tracing`? It's long, but we only call this once.

Otherwise, I'd provide a default implementation that delegates to `self.machine().context().tracing`.

Reply:
Added the default implementation, thanks. There are additionally 3 internal calls to this, and the many hops to get there adds coupling through those objects at more places.